src/resources/js/components/AccountEdit.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <v-container>
+
+  <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+  <h2>My Account</h2>
+
+  <v-form
+    ref="form"
+    v-model="valid"
+    lazy-validation
+  >
+    <v-text-field
+      v-model="updateUserInfo.fname"
+      :counter="10"
+      :rules="nameRules"
+      label="firstName"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.lname"
+      :counter="10"
+      :rules="nameRules"
+      label="lastName"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.email"
+      :rules="emailRules"
+      label="E-mail"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="updateUserInfo.password"
+      :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
+      :type="showPassword ? 'text' : 'password'"
+      label="password"
+      value=""
+      class="input-group--focused"
+      @click:append="showPassword = !showPassword"
+    ></v-text-field>
+
+    <v-btn
+      :disabled="!valid"
+      color="success"
+      class="mr-4"
+      @click="submitChange"
+    >
+      Submit
+    </v-btn>
+
+  </v-form>
+
+  </v-container>
+</template>
+
+<script>
+import axios from 'axios';
```

Comment:
it's a bit hs for this specific story, but since the other one got lost in the warp, don't forget to add the header with the bearer token for axios

Reply:
ok, added in [843a4](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/123/commits/843a4fcaa8a665292af7f3837f244bd112bab5ff)